src/cf-typography/src/licensed-fonts.less

```diff
@@ -0,0 +1,103 @@
+/*
+ * Source: http://fast.fonts.net/cssapi/44e8c964-4684-44c6-a6e3-3f3da8787b50.css
+ * This file has been edited to use absolute URLS so we can concatenate it with
+ * all of our other styles.
+ */
+@import url(//fast.fonts.net/t/1.css?apiType=css&projectid=44e8c964-4684-44c6-a6e3-3f3da8787b50);
+
+.use-font-net( @use-font-net ) when ( @use-font-net = 'true' ) {
```

Comment:
I think this mixin would be more appropriately titled something like `generate-font-face-rules`.